components/event-publisher-proxy/pkg/metrics/collector.go

```diff
@@ -11,16 +11,21 @@ const (
 	Errors = "event_publish_to_messaging_server_errors_total"
 	// Latency name of the latency metric
 	Latency = "event_publish_to_messaging_server_latency"
+	// EventType name of the eventType metric
+	EventType = "event_type_published"
```

Comment:
(nit) would it be better to have little more descriptive name `eventTypePubliushedMetricKey` and `eventTypePubliushedMetricHelp` ?